core/object/object.cpp

```diff
@@ -2086,15 +2086,17 @@ void ObjectDB::debug_objects(DebugFunc p_func) {
 	spin_lock.unlock();
 }
 
+#ifdef TOOLS_ENABLED
 void Object::get_argument_options(const StringName &p_function, int p_idx, List<String> *r_options) const {
+	String pf = p_function;
```

Comment:
```suggestion
	const String pf = p_function;
```
Same in other cases.

Reply:
Done so